src/beebop_deploy.py

```diff
@@ -107,13 +99,33 @@ def __init__(self, path, config_name=None, options=None):
         self.api_dbs_location = config.config_string(
             dat, ["api", "dbs_location"])
 
+        # acme
+
+        if self.use_acme:
+            acme_repo = config.config_string(
+                dat, ["acme_buddy", "image", "repo"])
+            acme_name = config.config_string(
+                dat, ["acme_buddy", "image", "name"])
+            acme_tag = config.config_string(
+                dat, ["acme_buddy", "image", "tag"])
+            self.acme_buddy_ref = constellation.ImageReference(
+                f"{self.registry}/{acme_repo}", acme_name, acme_tag)
```

Comment:
I'm a bit confused by this. `acme_repo` is already `ghcr.io/reside-ic`. Why are we prefixing it with a registry? Does this even work? Maybe we need to set `repo: reside-ic` instead?
 
Is this how we've done it in the other projects?

Reply:
Ah, I've slightly dumbly copied that from the pattern above that spins up the api container. `self.registry` is `config.config_string(dat, ["docker", "registry"])` which I think doesn't occur in any of the configs, so will come through as empty. 

So I could fix this up a bit...